packages/okta-angular/test/e2e/harness/package.json

```diff
@@ -22,7 +22,7 @@
     "@angular/platform-browser": "^4.2.4",
     "@angular/platform-browser-dynamic": "^4.2.4",
     "@angular/router": "^4.2.4",
-    "@okta/okta-angular": "file:../../../okta-okta-angular-0.0.8.tgz",
+    "@okta/okta-angular": "file:../../../okta-okta-angular-0.0.13.tgz",
```

Comment:
The changes here and below will go away once merged into the `1.0.0-angular` branch. They get regenerated when we run our tests.